### PR TITLE
Added DE locale (closes #236)

### LINKED ISSUE #236: German language support

### PR DESCRIPTION
**Pull request by DeMuu**
added translations and updated tests for DE locale

### CODE REVIEW

File: src/locale/de/build_format_locale/test.js [hunk 1]
```diff
@@ -0,0 +1,333 @@
+// @flow
+/* eslint-env mocha */
+
+var assert = require('power-assert')
+var buildFormatLocale = require('./')
+
+describe('en locale > buildFormatLocale', function () {
```
Comment (kossnocorp) [inline src/locale/de/build_format_locale/test.js]: ~~en~~ de

**Comment by DeMuu** [comment]
#237

File: src/locale/de/build_distance_in_words_locale/test.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
+// @flow
+/* eslint-env mocha */
+
+var assert = require('power-assert')
+var buildDistanceInWordsLocale = require('./')
+
+describe('en locale > buildDistanceInWordsLocale', function () {
```
Comment (kossnocorp) [inline src/locale/de/build_distance_in_words_locale/test.js]: Here is `en` too

**Comment by DeMuu** [comment]
Sorry I failed to see those beforehand. everything should be green now

**Comment by kossnocorp** [comment]
This was released as `1.7.0`, thank you for your help, much appreciated!

**Pull request merged**